app/src/main/java/io/pumpkinz/pumpkinreader/model/Comment.java

```diff
@@ -0,0 +1,37 @@
+package io.pumpkinz.pumpkinreader.model;
+
+import org.parceler.Parcel;
+
+import java.io.Serializable;
+import java.util.Date;
+
+
+@Parcel
+public class Comment implements Serializable {
+
+    String submitter;
```

Comment:
From https://github.com/johncarl81/parceler

If we make those variables private, then the only it can only access it by reflection, which is slow.